app/javascript/hooks/queries/recordings/useRoomRecordings.jsx

```diff
@@ -0,0 +1,11 @@
+import { useQuery } from 'react-query';
+import axios, { ENDPOINTS } from '../../../helpers/Axios';
+
+export default function useRoomRecordings(friendlyId) {
+  return useQuery('getRoomRecordings', () => axios.get(ENDPOINTS.room_recordings(friendlyId), {
+    headers: {
+      'Content-Type': 'application/json',
+      Accept: 'application/json',
+    },
```

Comment:
I think we don't need this since we are importing axiosInstance as default from the Axios helper?